src/test/java/ludogorie_soft/reservations_platform_api/controller/PropertyControllerIntegrationTest.java

```diff
@@ -78,16 +77,18 @@ void createProperty_ShouldReturnCreatedProperty() throws Exception {
         requestDto.setPetAllowed(false);
         requestDto.setPrice(150);
         requestDto.setPetRules("No pets allowed");
+        requestDto.setPropertyRules("Guests should maintain cleanliness");
```

Comment:
Can we use the `createDefaultPropertyRequestDto()` here?

Reply:
PropertyControllerIntegrationTest has been updated to use the PropertyTestHelper class